hatch_build.py

```diff
@@ -148,25 +148,20 @@
 
 DOC_EXTRAS: dict[str, list[str]] = {
     "doc": [
-        "astroid>=2.12.3,<3.0",
+        "astroid>=3",
         "checksumdir>=1.2.0",
         "click>=8.1.8",
-        # Docutils 0.17.0 converts generated <div class="section"> into <section> and breaks our doc formatting
-        # By adding a lot of whitespace separation. This limit can be lifted when we update our doc to handle
-        # <section> tags for sections
-        "docutils<0.17,>=0.16",
-        # The new theme 0.1.0 is for Sphinx 7. We should change it to >=0.1.0 when we move to Sphinx 7
-        "sphinx-airflow-theme>=0.0.12,<0.1.0",
+        "docutils>=0.21",
+        "sphinx-airflow-theme>=0.1.0",
+        "docutils>=0.21",
+        "sphinx-airflow-theme>=0.1.0",
```

Comment:
Dependency is doubled now - due to merge conflict...
```suggestion
        "docutils>=0.21",
        "sphinx-airflow-theme>=0.1.0",
```

Reply:
Thanks X2 :)